test/e2e/framework/format.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// FormatJSON is a helper that prints the inner object in json format.
+//
+// It can be used with log statements, for example:
+//  klog.Infof("object is %s", framework.FormatJSON(o))
```

Comment:
The purpose becomes clearer when using a different example:
```suggestion
//  klog.V(5).Infof("object is %s", framework.FormatJSON(o))
```